asr-9010.yaml

```diff
@@ -0,0 +1,45 @@
+manufacturer: Cisco
+model: ASR-9010
+slug: asr-9010
+part_number: ASR-9010
+u_height: 21
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco ASR 9000 Series Aggregation Services Routers Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/asr-9000-series-aggregation-services-routers/data_sheet_c78-501767.html)'
+power-ports:
+- name: 0/PT0-PM0
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM1
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM2
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT0-PM3
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM0
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM1
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM2
+  type: DC Terminal
+  maximum_draw: 2100
+- name: 0/PT1-PM3
+  type: DC Terminal
+  maximum_draw: 2100
+device-bays:
+- name: Slot 0 RSP 0
+- name: Slot 1 RSP 1
+- name: Slot 2 Line Card 0
+- name: Slot 3 Line Card 1
+- name: Slot 4 Line Card 2
+- name: Slot 5 Line Card 3
+- name: Slot 6 Line Card 4
+- name: Slot 7 Line Card 5
+- name: Slot 8 Line Card 6
+- name: Slot 9 Line Card 7
+- name: Slot 10 Line Card 8
```

Comment:
These should be modelled as module bays and not devices bays.

Please include a selection of line-cards and Sups as well if they don't already exist.